lib/logic/list-controller.js

```diff
@@ -92,14 +92,14 @@ ListController.prototype._loadAssignedPulls = function(prs) {
     for (var i = 0; i < prs.length; ++i) {
         var pr = prs[i];
         var me = pr.getAssignee();
-        if (pr.getOwner() === me) {
+        if (pr.getOwner().equals(me)) {
             // PR will appear in my pulls, don't duplicate it in assigned pulls
             continue;
         }
-        else if (pr.getLastCommenter() === pr.getOwner()) {
+        else if (pr.getOwner().equals(pr.getLastCommenter())) {
             assignedPulls.assignedNeedsAction.push(pr);
         }
-        else if (pr.getLastCommenter() !== me) {
+        else if (!me.equals(pr.getLastCommenter())) {
```

Comment:
Yes; we're only looking at assigned pull requests here, so we must the the assignee. And all pull requests must have an owner. Unless github is buggy.